### ISSUE TITLE
[Feature]: Release DB4S 3.13.1?

### ISSUE THREAD

**Issue by richb-hanover** [sqlitebrowser/sqlitebrowser#3724]
### Describe the new feature

There are lots of goodies in the nightly builds that could be sent to the world. These include:

- Everything that's in the Changelog
- Fix for macOS back to 10.13
- "Database currently busy" message
- Intellisense problem/GNOME Mutter
- Qt6 support
- and likely more

This could easily be enough good work to justify a new release. (Hopefully, the release process won't be nearly as difficult as last time.) Thanks

### Does this feature exist in another product or project? Please provide a link

N/A

### Do you have a screenshot? Please add screenshots to help explain your idea.

N/A

**Comment by justinclift** [comment]
Yeah, I think @lucydodo already has 3.13.1 roughly pencilled in for some time this month. :smile:

**Comment by lucydodo** [comment]
Yes, we are currently working on v3.13.1, which will include most of the changes currently in the `master` branch.
However, the Qt used for building will remain the current 5.x. An update to 6.x is planned for the next major version.

**Comment by richb-hanover** [comment]
I can confirm that 3.13.1-rc1 opens as expected on macOS Catalina (10.15)

**Comment by FriedrichFroebel** [comment]
The first `etc.` entry in the release notes does not look correct - at least considering the linked issue, but the commit not being linked as well.

**Comment by lucydodo** [comment]
@richb-hanover Thank you for your confirmation. :)

**Comment by lucydodo** [comment]
@FriedrichFroebel Thank you. I just fixed it. :)

**Comment by lucydodo** [comment]
v3.13.1 has been released: https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.13.1
